Examples/Framework/CMakeLists.txt

```diff
@@ -19,8 +19,8 @@ target_include_directories(
   PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>)
 target_link_libraries(
   ActsExamplesFramework
-  PUBLIC ActsCore ActsFatras Boost::boost ROOT::Core ROOT::Hist
-  PRIVATE Boost::filesystem dfelibs TBB::tbb)
+  PUBLIC ActsCore ActsFatras Boost::boost ROOT::Core ROOT::Hist TBB::tbb
```

Comment:
Needed to call a class member's destructor.

Reply:
Ok